runtime/src/bank.rs

```diff
@@ -294,6 +295,13 @@ impl BankRc {
     }
 }
 
+enum ProgramAccountLoadResult {
+    AccountNotFound,
+    InvalidAccountData,
```

Comment:
As this needs to handle the loader_v4 in the future as well, how about renaming these:
`ProgramAccount` => `ProgramOfLoader_v1_or_v2`
`ProgramDataAccount` => `ProgramOfLoader_v3`